sale_generator/views/sale_order_view.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="view_sale_order_tmpl_form" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_stock.view_order_form_inherit_sale_stock"/>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="is_template" invisible="1"/>
+            </field>
```

Comment:
Can you hide the field sale_order_template_id ?

Reply:
ok, but this is a field completely unrelated to this module (that field is declared in sale_management). Just to be sure, can you confirm ?

Reply:
confirmed and done, https://github.com/OCA/sale-workflow/pull/1079/commits/4d7a86854612182b8bffad909a4dee598b29f214